salt/journal-cms/config/etc-nginx-sites-enabled-journal-cms.conf

```diff
@@ -47,8 +52,8 @@ server {
         fastcgi_split_path_info ^(.+\.php)(/.+)$;
         include fastcgi_params;
         fastcgi_param SCRIPT_FILENAME $request_filename;
-        # https://github.com/elifesciences/issues/issues/4625#issuecomment-445829803
-        #fastcgi_param HTTP_X_CONSUMER_GROUPS $consumer_groups_filtered;
+        fastcgi_param HTTP_X_CONSUMER_GROUPS $consumer_groups_filtered;
+        fastcgi_param HTTP_AUTHORIZATION "";
```

Comment:
seems to be necessary to blank out `Authorization` to avoid `/ping` giving out a 403 on all requests with this header. But couldn't reproduce in production, I'll try to remove it.

Reply:
Tried to remove it, and got:
```
$ curl -v -H "X-Consumer-Groups: admin" -u "api_gateway_username:api_gateway_password" localhost:1241/ping -o /dev/null
* Connected to localhost (127.0.0.1) port 1241 (#0)
* Server auth using Basic with user 'api_gateway_username'
> GET /ping HTTP/1.1
> Host: localhost:1241
> Authorization: Basic YXBpX2dhdGV3YXlfdXNlcm5hbWU6YXBpX2dhdGV3YXlfcGFzc3dvcmQ=
> User-Agent: curl/7.47.0
> Accept: */*
> X-Consumer-Groups: admin
> 
< HTTP/1.1 403 Forbidden
< Server: nginx
< Content-Type: text/html; charset=UTF-8
< Transfer-Encoding: chunked
< Connection: keep-alive
< Vary: Accept-Encoding
< Cache-Control: must-revalidate, no-cache, private
< Date: Tue, 11 Dec 2018 08:45:30 GMT
< X-UA-Compatible: IE=edge
< Content-language: en
< X-Content-Type-Options: nosniff
< X-Frame-Options: SAMEORIGIN
< Expires: Sun, 19 Nov 1978 05:00:00 GMT
< Vary: 
< 
```
hence this is necessary.